db/migrate/20200504163928_create_tracks.rb

```diff
@@ -3,6 +3,7 @@ def change
     create_table :tracks do |t|
       t.string :slug, null: false, unique: true
       t.string :title, null: false
+      t.string :blurb, null: false, :limit => 400
```

Comment:
Use colon syntax: `limit: 400`

Reply:
Ah, I "borrowed" this from another file. Will update